### ISSUE TITLE
Giving error when we run yarn test

### ISSUE THREAD

**Issue by mayank-aika** [lemurheavy/coveralls-public#1583]
/var/www/projects/ui-command-contacts/node_modules/coveralls/bin/coveralls.js:19
      throw err;
      ^
Bad response: 405 <html>
<head><title>405 Not Allowed</title></head>
<body>
<center><h1>405 Not Allowed</h1></center>
<hr><center>nginx</center>
</body>
</html>

(Use `node --trace-uncaught ...` to show where the exception was thrown)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Current version**: "^3.1.0"

**Comment by felipecmachado** [comment]
The database maintenance has ended but now we are getting a new error `This website is under heavy load (queue full)`.
It seems all systems are operational, according to the https://status.coveralls.io.

Is there any news?

**Comment by afinetooth** [comment]
@mayank-aika, @felipecmachado, apologies for the inconvenience.

### tl;dr:
We are back in maintenance mode right now, with hopes of returning to production in the next hour. However, there's a chance we'll need to remain in maintenance for the bulk of today, in which case, we recommend removing the coveralls steps from your non-crucial CI builds until later today.

Updates posted here:
https://status.coveralls.io/

Feel free to check in here as well.

### More detail:
We took the site out of maintenance mode at about 3am this morning at about 95% complete, but then transactions slowed and started failing. It turns out one index on our largest table is still rebuilding, or perhaps failing---in any case, was taking very very long to rebuild, so we put the site back in maintenance mode to let it finish or troubleshoot.

We have our Postgres consultants and AWS resources on it, so hopefully we'll be able to take it back out of maintenance soon, but the word I have now is that we could be down for most of today as the large index rebuilds. We might be able to rebuild it during production if we complete a Postgres upgrade, which is what the team is looking at right now.

In the scenario where we're down most of today, you might want to disable coveralls steps for any non-crucial CI builds until later today.

---

### Related:
- https://github.com/lemurheavy/coveralls-public/issues/1582
